README.md

```diff
@@ -37,8 +37,24 @@ To check whether any format violation exists, run `spotlessCheck` gradle task. I
 
 Both plugins are enabled on gradle's `compileJava` task.
 
-## References
+### Testing
+
+Per best practice, we maintain separate directories for unit and integration testings:
+
+- [unit test](src/test)
+- [integration test](src/integrationTest)
+
+Integration tests will connect to a MongoDB deployment. You may change the default [MongoDB connection string](https://www.mongodb.com/docs/manual/reference/connection-string/) configured as below at [hibernate.properties](src/integrationTest/resources/hibernate.properties):
 
+```properties
+jakarta.persistence.jdbc.url=mongodb://localhost/mongo-hibernate-test?directConnection=false
+```
+
+### CI/CD
+An internal CI/CD pipeline is based on an open-source project [evergreen](https://github.com/evergreen-ci/evergreen), a distributed continuous integration system from MongoDB. The corresponding evergreen configuration resources reside in a [.evergreen](/.evergreen) directory under the project root folder.
+
+## References
+It would be highly helpful to refer to the following awesome articles or resources when need arises:
```

Comment:
```suggestion
```
I don't think we need to comment on this - if the documents have been included under references in the main readme, they must be helpful and good.